app/frontend/src/service/user.test.ts

```diff
@@ -0,0 +1,128 @@
+import { Empty } from "google-protobuf/google/protobuf/empty_pb";
```

Comment:
Added some more unit tests on the service layer, mainly to check the values are set correctly before it gets sent to the grpc client, which are still mocked. I thought this is still worthwhile as there's still a little bit of logic in these and weren't tested in the reducer tests as this layer has been mocked out in those.